src/qibolab/native.py

```diff
@@ -64,7 +64,16 @@ def from_dict(cls, name, pulse, qubit):
         """
         kwargs = pulse.copy()
         kwargs["pulse_type"] = PulseType(kwargs.pop("type"))
-        return cls(name, **kwargs, qubit=qubit)
+        kwargs["qubit"] = qubit
+        return cls(name, **kwargs)
+
+    @property
+    def to_dict(self):
```

Comment:
```suggestion
    def raw(self):
```

As already said, that's the only name I came with, but if you have a more descriptive one even better